lib/android/audio.nit

```diff
@@ -31,37 +31,49 @@ in "Java" `{
 	import android.media.SoundPool;
 	import java.io.IOException;
 	import android.media.AudioManager;
+	import android.media.AudioManager.OnAudioFocusChangeListener;
 	import android.content.Context;
 	import android.util.Log;
 `}
 
+in "Java inner" `{
+	static OnAudioFocusChangeListener afChangeListener = new OnAudioFocusChangeListener() {
+		public void onAudioFocusChange(int focusChange) {
+			if(focusChange == AudioManager.AUDIOFOCUS_LOSS_TRANSIENT) {
+			}else if (focusChange == AudioManager.AUDIOFOCUS_GAIN) {
+			}else if (focusChange == AudioManager.AUDIOFOCUS_LOSS) {
+			}
+		}
+	};
+`}
+
 # AudioManager of the application, used to manage the audio mode
 extern class NativeAudioManager in "Java" `{ android.media.AudioManager `}
 	super JavaObject
 
+	# Returns the current audio mode ( MODE_NORMAL = 0, MODE_RINGTONE = 1, MODE_IN_CALL = 2 or MODE_IN_COMMUNICATION = 3 )
 	fun mode: Int in "Java" `{ return recv.getMode(); `}
+
+	# Sets the audio mode ( MODE_NORMAL = 0, MODE_RINGTONE = 1, MODE_IN_CALL = 2 or MODE_IN_COMMUNICATION = 3 )	
 	fun mode=(i: Int) in "Java" `{ recv.setMode((int)i); `}
-	fun wired_headset_on: Bool in "Java" `{ return recv.isWiredHeadsetOn(); `}
-	fun wired_headset_on=(b: Bool) in "Java" `{ recv.setWiredHeadsetOn(b); `}
-	fun speakerphone_on: Bool in "Java" `{ return recv.isSpeakerphoneOn(); `}
-	fun speakerphone_on=(b: Bool) in "Java" `{ recv.setSpeakerphoneOn(b); `}
-	fun manage_audio_mode in "Java" `{
-		recv.setMode(0);
-		if (recv.isWiredHeadsetOn()) {
-			recv.setSpeakerphoneOn(false);
-		} else {
-			recv.setSpeakerphoneOn(true);
-		}
-	`}
 
+	# Send a request to obtain audio focus
```

Comment:
Why use the third person in the previous comment and the imperative here? Please be consistent even if you are an intern :p